src/_locales/en/messages.json

```diff
@@ -0,0 +1,109 @@
+{
+  "launchInExternalBrowser": {
+    "message": "Open page in $browser$",
+    "placeholders": { "browser": { "content": "$1" } }
+  },
+  "launchInExternalBrowserLink": {
+    "message": "Open link in $browser$",
+    "placeholders": { "browser": { "content": "$1" } }
+  },
+  "welcomeHeading": {
+    "message": "Welcome to Firefox Launch"
```

Comment:
I don't know where we are with the content ([open ticket](https://mozilla-hub.atlassian.net/browse/LR-62)). In case this needs to stay in English, we'll need a comment to explain it to localizers.

Reply:
PR: https://github.com/mozilla/firefox-launch/pull/21